web/package.json

```diff
@@ -66,7 +66,7 @@
     "eject": "react-scripts eject",
     "cy:open": "cypress open",
     "cy:run": "cypress run",
-    "cy:ci": "cypress run --parallel --record --key $CYPRESS_RECORD_KEY",
+    "cy:ci": "cypress run --record --key $CYPRESS_RECORD_KEY",
```

Comment:
Why did you remove the `--parallel` flag?

Reply:
just testing

Reply:
right now i am isolating the stable tests from the one that fail all the time.
thats why I removed the parallel.